.github/workflows/ci.yaml

```diff
@@ -0,0 +1,138 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - _integration/**/*
+      - test_consume_workiva/**
+    tags:
+      - "**"
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  build-image:
+    name: Build Image
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk: [2.19.6]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: dart-lang/setup-dart@v1.6.2
+        with:
+          sdk: ${{ matrix.sdk }}
+      - name: Install tools
+        run: |
+          sudo apt-get update -qq && \
+          sudo apt-get dist-upgrade -y && \
+          sudo apt-get install -y jq && \
+          sudo apt-get autoremove -y && \
+          sudo apt-get clean all
```

Comment:
```suggestion
```
We don't need to do any of this in GHA anymore. We can remove the whole build-image job